### ISSUE TITLE
Suggestion: Implement Request for Packaging Queue

### ISSUE THREAD

**Issue by oxr463** [emscripten-core/emscripten#10048]
See: https://gitlab.com/fdroid/rfp

I think this would be a big improvement on the current emscripten-ports process.

**Comment by kripken** [comment]
Looking at the approach in that link, there does seem to be a lot we could learn from there, thanks @oxr463 !

**Comment by oxr463** [comment]
- [ ] Create a new repository, <https://github.com/emscripten-core/emscripten-ports>
  - [ ] README.md - highlight process for requesting a port, refer to documentation
  - [ ] Create issue templates
    - [ ] New Package
    - [ ] Existing Package

**Comment by sbc100** [comment]
I think the main problem is that our ports system is currently very limited, and not generally open to new packaged.  There have been some plans made to impove this and design some kind of "contrib" section where we can add new ports, but I think think we want to grow the core set of ports, especially not without first improving the ports system itself.

**Comment by sbc100** [comment]
BTW we already have an entire github org called https://github.com/emscripten-ports

**Comment by oxr463** [comment]
> 
> 
> BTW we already have an entire github org called https://github.com/emscripten-ports

Yes, but the process for upstreaming a new port is not very clear. If a repository for this effort was made, then we could track this via issues, separate from the emscripten issue tracker.

**Comment by oxr463** [comment]
This is not stale.
